docs/developer/advanced/upgrading-nodejs.asciidoc

```diff
@@ -0,0 +1,76 @@
+[[upgrading-nodejs]]
+== Upgrading Node.js
+
+{kib} requires a specific Node.js version to run.
+When running {kib} from source, you must have this version installed locally.
+
+The required version of Node.js is listed in several different files throughout the {kib} source code.
+Theses files must be updated when upgrading Node.js:
+
+  - {kib-repo}blob/{branch}/.ci/Dockerfile[`.ci/Dockerfile`] - The version is specified in the `NODE_VERSION` constant.
+    This is used to pull the relevant image from https://hub.docker.com/_/node[Docker Hub].
```

Comment:
There is an extra / between .com and node.
https://hub.docker.com//node[Docker Hub]

Also from "Note that Docker Hub" through "just" is italicized.  I think you might want to use regular font here.

Reply:
> There is an extra / between .com and node.
> https://hub.docker.com//node[Docker Hub]

Not sure what you mean? The link is written as `https://hub.docker.com/_/node` in the asciidoc file - notice the extra `_` (underscore)

Reply:
> Also from "Note that Docker Hub" through "just" is italicized. I think you might want to use regular font here.

Addressed in #87976